src/entity/Cell.js

```diff
@@ -75,8 +76,7 @@ Cell.prototype.getSpeed = function() {
     // Old formulas:
     // return 5 + (20 * (1 - (this.mass/(70+this.mass))));
     // return this.gameServer.config.playerSpeed * Math.pow(this.mass, -0.22) * 50 / 40;
-    var t = Math.PI * Math.PI;
-    return this.gameServer.config.playerSpeed * Math.pow(this.mass, -Math.PI / t / 1.5);
+    return this.gameServer.config.playerSpeed * Math.pow(this.mass, -Math.PI / (Math.PI * Math.PI) / 1.5);
```

Comment:
Math.PI / Math.PI = 1 and 1 / 1.5 = 0.667 :wink: 

So you can write `-1 / Math.PI * 0.667` - it will be faster